velero/minio/20-statefulset.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  namespace: velero
+  name: minio
+  labels:
+    component: minio
+spec:
+  selector:
+    matchLabels:
+      component: minio
+  serviceName: minio
+  template:
+    metadata:
+      labels:
+        component: minio
+    spec:
+      volumes:
+      - name: storage
+        emptyDir: {}
+      - name: config
+        emptyDir: {}
+      securityContext:
+        runAsUser: 0
+        fsGroup: 999
+      containers:
+      - name: minio
+        image: minio/minio:latest
```

Comment:
We have to be careful when using :latest tag. If something changes over time and latest is not backwards compatible (i.e env vars, args in the command, etc) the use case stops working. I would pin it down to the current latest tag.